public/js/Chart.min.js

```diff
@@ -1,14 +0,0 @@
-/*!
```

Comment:
I'm pretty sure we aren't charting or anything yet, but please be sure this isn't needed.